content/_includes/site/footer.njk

```diff
@@ -1,10 +1,21 @@
 {% import 'contact.macros.njk' as contact %}
 {% import 'utility.macros.njk' as utility %}
 
+{% set add_cta = page.url | addCallToAction %}
+{% if add_cta %}
+    {% include './cta.njk' %}
+{% endif %}
+
 <div
   data-layout="footer">
   {% if page.url != '/contact/' %}
     <aside aria-label="contact">
+      {% if not add_cta %}
+        <div class="contact-form-intro">
+          <h3>Let's chat about your web project</h3>
+          Fill out the form, <a href="https://calendly.com/oddbird/discovery"> schedule a call</a>, <a href="mailto:birds@oddbird.net"> email</a>, <a href="https://twitter.com/oddbird"> tweet</a>, or find us on <a href="https://github.com/oddbird">GitHub</a>.
```

Comment:
'tweet' stands out here grammatically. Find us on twitter or github? Send us an email or tweet? Not sure what the best fix is.

Reply:
Good call - I think I'll adjust it to read: 
`Fill out the form, schedule a call, send us an email, or find us on twitter or GitHub.`
Does that work for you @SondraE ?

Reply:
+1